server/handler/ldif/index.go

```diff
@@ -22,25 +22,32 @@ var indexAttributes = map[string]string{
 	"gidNumber":    "eq",
 	"mail":         "eq",
 	"memberUid":    "eq",
+	"o":            "eq",
 	"ou":           "eq",
 	"uid":          "eq",
 	"uidNumber":    "eq",
 	"uniqueMember": "eq",
 
-	"kopanoAccount":       "eq",
```

Comment:
does these indexes then need to be re-added in a Kopano specific build?

Reply:
Yes